src/translations/sk.json

```diff
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "logged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
+    "logging_out": "Odhlásenie",
+    "sign_up": "Zaregistrovať sa",
+    "signing_up": "Registrácia",
+    "forgot_password": "Zabudli ste heslo?",
+    "recover_password": "Obnoviť heslo",
+    "send_recovery_email": "Odoslať e-mail pre obnovenie",
+    "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+    "never_mind": "Nevadí",
+    "update_password": "Aktualizovať heslo",
+    "updated_password": "Aktualizácia heslá",
+    "complete_your_signup": "Dokončite registráciu",
+    "site_url_title": "Nastavenie vývoja",
+    "site_url_link_text": "Vymazať URL localhost",
+    "site_url_message": "Zdá sa, že používate lokálny server. Povedzte nám adresu URL svojho Netlify servera.",
+    "site_url_label": "Zadajte URL svojho servera Netlify",
+    "site_url_placeholder": "URL vášho Netlify servera",
+    "site_url_submit": "Nastaviť adresu URL",
+    "message_confirm": "Na váš e-mail bol odoslaný odkaz na potvrdenie, pokračujte kliknutím na tento odkaz.",
+    "message_password_mail": "Poslali sme vám e-mail pre obnovenie hesla, heslo obnovíte kliknutím na odkaz v e-mailu.",
+    "message_email_changed": "Vaša e-mailová adresa bola aktualizovaná!",
+    "message_verfication_error": "Pri overovaní vášho účtu došlo k chybe. Prosím, skúste to znova alebo kontaktujte správcu.",
+    "message_signup_disabled": "Registrácia pre verejnosť sú zakázané. Kontaktujte správcu a požiadajte o pozvánku.",
+    "form_name_placeholder": "Meno",
+    "form_email_label": "Zadajte svoj e-mail",
+    "form_name_label": "Zadajte svoje meno",
+    "form_email_placeholder": "E-mail",
+    "form_password_label": "Zadajte svoje heslo",
+    "form_password_placeholder": "Heslo",
+    "coded_by": "Vytvorené Netlify",
+    "continue_with": "Pokračovať ako",
```

Comment:
Honestly, I'm not sure of this one, as it's a really difficult concept to translate in little words as possible, but I think something to indicate "Continue _using_ X" might sound a bit better here. What do you think?

```suggestion
    "continue_with": "Pokračovať cez",
```

Reply:
@iKristy How about "Pokračovať pomocou" (not sure if it is correct Slovak, please correct :)). Here is a Czech example

![image](https://user-images.githubusercontent.com/1215304/120591331-42a31d00-c43c-11eb-9740-e9fd9e5c2bd5.png)

Reply:
Oh, 'pomocí' / 'pomocou' is not bad, actually!

An initial example I looked at for using 'cez' for more context why I suggested it:

**Martinus:**
![image](https://user-images.githubusercontent.com/7912948/120669293-385a4080-c487-11eb-9eb9-7b1dbfbff111.png)

After looking for more examples just to see what's the best practice for this what other services use:

**Alza:**
![image](https://user-images.githubusercontent.com/7912948/120669585-7d7e7280-c487-11eb-83d2-9830df785344.png)
![image](https://user-images.githubusercontent.com/7912948/120669712-9be46e00-c487-11eb-9a06-a6e9a7dbf73b.png)

**Refresher.sk:**
![image](https://user-images.githubusercontent.com/7912948/120678608-2fba3800-c490-11eb-8cb7-7f58ae994545.png)

**Mall.cz:**
![image](https://user-images.githubusercontent.com/7912948/120678750-57110500-c490-11eb-8110-5e51dada0df1.png)

...I think either works tbh 😅  But looks like 'cez' / 'přes' might be more commonly used 🤔 

So I'll leave this one up to you to decide! 🙇‍♀️

Reply:
@iKristy Alright, I will use přes / cez.

Reply:
Thank you 🙌